rustls-bench/src/main.rs

```diff
@@ -832,16 +833,96 @@ impl Options {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, ValueEnum)]
+enum Provider {
+    None,
```

Comment:
This was needed when built with no features ("references are always considered inhabited") but actually making the receiver `self` rather than a reference fixes that.